lotus/sem_ops/postprocessors.py

```diff
@@ -52,6 +57,28 @@ def map_postprocess(llm_answers: list[str], cot_reasoning: bool = False) -> Sema
     return SemanticMapPostprocessOutput(raw_outputs=llm_answers, outputs=outputs, explanations=explanations)
 
 
+def extract_postprocess(llm_answers: list[str]) -> SemanticExtractPostprocessOutput:
+    """
+    Postprocess the output of the schema operator to extract the schema.
+
+    Args:
+        llm_answers (list[str]): The list of llm answers containging the schema.
+
+    Returns:
+        SemanticSchemaPostprocessOutput
```

Comment:
Replace all instances of `SemanticSchemaPostprocessOutput` with `SemanticExtractPostprocessOutput`